.github/workflows/main.yml

```diff
@@ -160,7 +160,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install -U pip
+          pip install -U pip==24.2
```

Comment:
This is in the `test` job, but it's needed in the  `check`  job (where the "Check requirements files" step is")